internal/aez/heater_regulators.go

```diff
@@ -61,18 +61,19 @@ type HTR struct {
 
 //HTRReport housekeeping report returns data on all heater regulators in useful units.
 type HTRReport struct {
-	HTR1A  float64 // Heater 1 Temperature sense A ⁰C
-	HTR1B  float64 // Heater 1 Temperature sense B ⁰C
-	HTR1OD float64 // Heater 1 Output Drive setting voltage
-	HTR2A  float64
-	HTR2B  float64
-	HTR2OD float64
-	HTR7A  float64
-	HTR7B  float64
-	HTR7OD float64
-	HTR8A  float64
-	HTR8B  float64
-	HTR8OD float64
+	HTR1A    float64 // Heater 1 Temperature sense A ⁰C
+	HTR1B    float64 // Heater 1 Temperature sense B ⁰C
+	HTR1OD   float64 // Heater 1 Output Drive setting voltage
+	HTR2A    float64
+	HTR2B    float64
+	HTR2OD   float64
+	HTR7A    float64
+	HTR7B    float64
+	HTR7OD   float64
+	HTR8A    float64
+	HTR8B    float64
+	HTR8OD   float64
+	WARNINGS []error
```

Comment:
It would kind of be nicer if this was just an `error` or a `string` here in the end else each line will be `[]` for no warnings.

Reply:
Not sure I see why that would be an improvement, but I may be misunderstanding your point. As I see it, this struct is an internal data exchange format ("model"), and the representation ("view") is up to the function that does something with it. For instance, a logger may want to filter or compound errors.

Reply:
Well error will be `[]` if we go with this for when there's no error.

Reply:
Ah, are you suggesting I adapt the CSV-writer to handle a slice?

Reply:
well right now you can't since we're in a race-condition, but it is a bit annoying that we have to introduce a special case. Currently the code that seralizes the HTRReport has no clue about the contents of the struct. We will have to add a special case for it. Maybe that's better than making the WARNINGS a string. I don't know.

Reply:
Ok, then I see the problem. I think that this is the wrong place to put logic for what to do with the error values, but I guess it's a matter of overall structure. This struct could alternatively be seen as "ready for export", but my feeling is that this would tie it too closely to the CSV exporter use case. Feel free to try to convince me if you disagree.